### PR TITLE
2020-08-28 v. 1.0.8: added distribute flavors

### LINKED ISSUE #79: Create `default` and `googlePlay` flavors
Create flavors and separate Google Play specific logic (in-app updates) by flavors (in-app updated should be only in `googlePlay` flavor).

### PR DESCRIPTION
**Pull request by vipozdn**
1. Added:
- app/src/main/... MainActivity class;
- app/src/thirdpartyStores/... MainActivity class.

2. Updated:
buildd.gradle (app) file
- added flavors,
- Play Core Library depenency changed to only googlePlay-flavor implementation.

3. Deleted:
- unused Play Core Library imports in app/src/thirdpartyStores/... MainActivity class.

## Checklist

__Common__

- [x] I am ran the app before creating PR
- [x] I am ran all tests before creating PR

__UI__

- [x] I am ran the app for visual checks.

__Logic__

- [x] I am tested basic app functionality.

## Changes

Describe all changes here:

- First;
- Second;
- Third;
- ...

## Comments

Describe all additional information here.

### CODE REVIEW

File: app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import android.content.IntentSender;
 import android.net.Uri;
 import android.os.Bundle;
+import android.util.Log;
```
Comment (fartem) [inline app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java]: [Codacy](https://app.codacy.com/gh/hash-checker/hash-checker-lite/pullRequest?prid=7989232) find issue here. Please, fix it.

**Review by fartem — CHANGES_REQUESTED** [review]
Check [Codacy](https://app.codacy.com/gh/hash-checker/hash-checker-lite/pullRequest?prid=7989232) issues.

File: app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java (hunk 1 above)
Comment (vipozdn) [reply app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java]: done ✓

**Review by fartem — APPROVED** [review]

**Comment by fartem** [comment]
@vipozdn thanks.

**Pull request merged**